examples/hosting-attachedDomain/createAttachedDomain.php

```diff
@@ -0,0 +1,82 @@
+<?php
+require __DIR__ . '/vendor/autoload.php';
+use \Ovh\Api;
+use GuzzleHttp\Client;
+
+// Informations about your application
+$applicationKey = "your_app_key";
+$applicationSecret = "your_app_secret";
+$consumer_key = "your_consumer_key";
+
+// Information about API and rights asked
+$endpoint = 'ovh-eu';
+
+// Information about your web hosting and the attached domain (compulsory)
+$domain = 'mydomain.ovh'; // Web hosting id (often domain order with it)
+$domainToAttach = 'myotherdomaintoattach.ovh';
+$path = 'otherFolder'; // Folder where website files are stored
+
+// Informations about the attached domain (optional)
+$cdn = 'none'; // Enable CDN on the attached domain. Can be NULL, 'none' or 'active'
+$firewall = 'none'; // Enable HTTP firewall (block dangerous pattern requests with Apache mod_security). Can be NULL, 'active' or 'none'
+$ownLog = 'myotherdomaintoattach.ovh'; // Separate logs by domain. Can be 'null' or domain offer you have at OVH
+
+$http_client = new Client([
+	'timeout'         => 30,
+	'connect_timeout' => 5,
+]);
+
+// Create a new attached domain
+$conn = new Api(    $applicationKey,
+                    $applicationSecret,
+                    $endpoint,
+                    $consumer_key,
+		    $http_client);
+
+try {
+
+	// This call will create a "task". The task is the status of the attached domain creation.
+	// You can follow the task on /hosting/web/{serviceName}/tasks/{id}
+	$task = $conn->post('/hosting/web/' . $domain . '/attachedDomain', array(
+		'domain'	=> $domainToAttach,
+		'path'		=> $path,
+		'cdn'		=> $cdn,
+		'firewall'	=> $firewall,
+		'ownLog'	=> $ownLog,
+	));
+
+	echo "Task #" . $task['id'] . " is created" . PHP_EOL;
+
+	// we check each 5 seconds if task are ended
+	// When the task disappear, the task is done
+	while ( 1 ) {
+		try {
+			$wait = $conn->get('/hosting/web/' . $domain . '/tasks/' . $task['id']);
+			
+			if ( strcmp( $wait['status'], 'error' ) === 0 ) {
+				// The task is in error state. Please check your parameters, retry or contact support.
```

Comment:
I can afford telling for every single lines, but your comments are really helpful. That's a really good work. Thanks!